metadata_backend/api/auth.py

```diff
@@ -136,8 +134,36 @@ async def callback(self, req: Request) -> Response:
                 reason="Could not set user, missing claim CSCUserName, remoteUserIdentifier or sub."
             )
 
+        # Handle projects, they come in different formats depending on the service used.
+        # Current project sources: CSC projects, LS AAI groups
+        projects: List[Dict[str, str]] = []
+        if "sdSubmitProjects" in session["userinfo"]:
+            # CSC projects come in format "project1 project2"
+            csc_projects = session["userinfo"]["sdSubmitProjects"].split(" ")
+            for csc_project in csc_projects:
+                projects.append(
+                    {
+                        "project_name": csc_project,
+                        "origin": "csc",
+                    }
+                )
+        if "eduperson_entitlement" in session["userinfo"]:
+            # LS AAI groups come in format ["group1", "group2"]
+            for group in session["userinfo"]["eduperson_entitlement"]:
+                projects.append(
+                    {
+                        "project_name": group,
```

Comment:
Shouldn't we remove the `#`, and what comes after, as that is not part of the name of the group?

Reply:
I'm okay with removing that, as it's not important to the user